config/locales/en.yml

```diff
@@ -46,6 +46,9 @@ en:
       #################################################################
     not_internet_connection_error: 'Error: Not internet connection. Please by connecting to the internet'
   commands:
+    _setup:
```

Comment:
What does it mean by starting with an underscore?

I guess it would be better to move this section up to the upper level and define `errors` section like the following.

```
  errors:
    check_update_failure: ...
  commands: ...
```

Reply:
thanks you for comment!

i wanted to show like private method...

sound good! i try it!

Reply:
@jmatsu 
i fixed. 
please review!